monai/data/wsi_reader.py

```
@@ -210,7 +210,7 @@ def get_valid_level(
                 # Set the default value if no resolution parameter is provided.
                 level = 0
             if level >= n_levels:
-                raise ValueError(f"The maximum level of this image is {n_levels-1} while level={level} is requested)!")
+                raise ValueError(f"The maximum level of this image is {n_levels - 1} while level={level} is requested)!")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)